fjord/l10n/utils.py

```diff
@@ -0,0 +1,9 @@
+import re
+
+
+_COLLAPSE_RE = re.compile(r'\s+', re.UNICODE)
+
+
+def collapse_whitespace(message):
+    """Collapses consecutive whitespace into a single space"""
+    return _COLLAPSE_RE.sub(' ', message).strip()
```

Comment:
`return u' '.join(map(lambda s: s.strip(), message.strip().splitlines()))` seems slightly faster and equivalent, but do your own testing.

```
 ~ python -mtimeit -s "message = u'  tést \n\t foo \nyo'" "u' '.join(map(lambda s: s.strip(), message.strip().splitlines()))"
1000000 loops, best of 3: 1.64 usec per loop
 ~ python -mtimeit -s "import re; message = u'  tést \n\t foo \nyo'; _COLLAPSE_RE = re.compile(r'\s+', re.UNICODE)" "_COLLAPSE_RE.sub(' ', message).strip()"
1000000 loops, best of 3: 1.96 usec per loop
```